### DISCUSSION TITLE
LEFT BORDER ONLY NOT WORKING:  border border-l-4 border-indigo-500

### DISCUSSION THREAD

**Discussion by AllisonBarnett** [tailwindlabs/tailwindcss#8723] [Help]
My class is "border border-l-4 border-indigo-500"

I just want a line on the left side of my content, but I can't get this to show up with out having a border around the whole section. Is it even possible in Tailwind to only have a border on the left side show up?

**Comment by crswll** [comment]
You want to remove `border`. `border` sets it on all 4 sides. `border-l-*` sets it for just the left.
